lib/ltproject.coffee

```diff
@@ -0,0 +1,57 @@
+path = require 'path'
+fs = require 'fs'
+{Directory} = require 'atom'
+
+module.exports =
+
+class LTProject
+  constructor: ->
+    console.log 'Create projct settings'
+
+    @TEXroot = null
+
+    # finding .lt-projet file in project path
+    buffer_path = path.dirname(atom.workspace.getActiveTextEditor().getPath())
```

Comment:
It's probably better to check the paths in `atom.project.getPaths()` perhaps for the first `.lt-project` file. Otherwise, won't this break when the active text editor is in a subfolder of the main project?